### ISSUE TITLE
Treelist Stylesheet

### ISSUE THREAD

**Issue by wylie** [ebsco/edna#93]
This stylesheet, `less/_treelist.less`, needs some cleanup. Remove the `foo: bar` and commented out lines.

**Comment by wylie** [comment]
I'm thinking that the treelist needs to be seriously worked on. Our stylesheet currently has this structure

``` css
.treelist {
    .list {}
}
```

I think that we should combine the two and make the treelist an actual list. The `.treelist` class actually has no styles tied to it, so this could be a simple task of combining the classes (`.list.treelist` as opposed to `.treelist .list`) and revising the template.

Then we could remove this stylesheet, by merging it into the lists stylesheet, or keep it as a way to differentiate the different types of lists we use.

For now I have removed some commented out declarations and the `foo: bar;` business.

**Comment by gjjones** [comment]
Is a treelist something we'd even want in edna? My first thought is to have edna cognizant of single layer lists, and have nests lists play friendly. But to explicitly state that UI pattern is a tree list makes me think more of a buzz widget than an edna component.

**Comment by wylie** [comment]
yep, this should live in Buzz. Once we go atomic it definitely will, but the base list styles will be in Edna.

**Comment by gjjones** [comment]
:+1:
